org.aegisub.Aegisub.yml

```
@@ -31,6 +31,13 @@ add-extensions:
     bundle: true
     no-autodownload: false
     autodelete: true
+  org.aegisub.Aegisub.Automations.DependencyControl:
+    directory: automations/dependency_control
+    subdirectories: true
+    add-ld-path: lib
+    merge-dirs: share/aegisub/automation/autoload;share/aegisub/automation/include;share/aegisub/automation/demo
+    no-autodownload: true
+    autodelete: true
```

```suggestion
  org.aegisub.Aegisub.Automations.DependencyControl:
    directory: automations/dependency_control
    add-ld-path: lib
    no-autodownload: true
    autodelete: true
```

`subdirectories` and `merge-dirs` shouldn’t be needed here.